src/libraries/System.Private.CoreLib/src/System/Globalization/CompareInfo.WebAssembly.cs

```diff
@@ -0,0 +1,67 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+
+namespace System.Globalization
+{
+    internal static unsafe class CompareInfoInterop
```

Comment:
Wouldn't it better to separate JS calls like it is done now for Unmanaged code calls https://github.com/dotnet/runtime/blob/main/src/libraries/Common/src/Interop/Interop.Casing.cs ?

Reply:
You're right, we even have a special `Browser` directory there where I can place the interop calls.